compiler/tflchef/core/src/OpChef.h

```diff
@@ -45,4 +45,14 @@ struct OpChefFactory
   virtual std::unique_ptr<OpChef> create(const tflchef::Operation *operation) const = 0;
 };
 
+static void CheckCustomOp(const tflchef::Operation operation, std::string op_type)
+{
+  if ((operation.has_extype() && operation.extype() == "Custom") ||
+      (operation.has_type() && operation.type() == "Custom"))
+  {
+    assert(operation.has_custom_code());
+    assert(operation.custom_code() == op_type);
+  }
+};
+
```

Comment:
I created a function here to avoid duplication.

Reply:
Q) Can you explain more about what you mean by `avoid duplication` ?